tests/test_backups.py

```diff
@@ -0,0 +1,171 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+import pathlib
+import subprocess
+
+import conftest
+import pytest
+from docker.models import containers
+
+log = logging.getLogger(__file__)
+log.setLevel("DEBUG")
+
+CAPELLA_VERSION = os.getenv("CAPELLA_VERSION", "")
+
+
+def is_capella_5_x_x() -> bool:
+    return CAPELLA_VERSION in ("5.0.0", "5.2.0")
+
+
+def is_capella_6_x_x() -> bool:
+    return CAPELLA_VERSION in ("6.0.0")
+
+
+@pytest.fixture(name="local_git_container")
+def fixture_local_git_container() -> containers.Container:
+    ports: dict[str, int] = {"80/tcp": 9010}
+
+    with conftest.get_container(
+        image="tests/local-git-server", ports=ports
+    ) as container:
+        conftest.wait_for_container(container, "server started")
+        yield container
+
+
+@pytest.fixture(name="t4c_backup_container")
+def fixture_t4c_backup_container(request) -> containers.Container:
```

Comment:
I thought that adding it as a dependency of the fixture would not work (since I provide arguments from the parameterized test). But luckily it's not a problem, so I moved it to the correct place